packages/types/src/codec/create/sanitize.ts

```diff
@@ -5,7 +5,7 @@
 
 type Mapper = (value: string) => string;
 
-const ALLOWED_BOXES = ['Compact', 'DoubleMap', 'Linkage', 'Result', 'Option', 'Vec'];
+const ALLOWED_BOXES = ['BTreeMap', 'Compact', 'DoubleMap', 'Linkage', 'Result', 'Option', 'Vec'];
```

Comment:
This is the one that will allow `BTreeMap<Text, u32>` to actually come back unmangled (unlike the currect where the type args are removed)